tests/common.py

```diff
@@ -173,3 +172,47 @@ def create_company(cls, name, **kwargs):
         }
         vals.update(kwargs)
         return Company.create(vals)
+
+    @classmethod
+    def create_simple_inbound_route(cls, picking_type_internal):
+        Route = cls.env['stock.location.route']
+        Path = cls.env['stock.location.path']
+        Sequence = cls.env['ir.sequence']
+        Rule  = cls.env['procurement.rule']
+
+        route_vals = {
+            "name": "Putaway",
+            "sequence": 10,
+            "product_selectable": False,
+            "warehouse_selectable": True,
+            "warehouse_ids": [(6, 0, [picking_type_internal.warehouse_id.id])]
+        }
+        route = Route.create(route_vals)
+
+        # PUTAWAY
+        sequence_putaway = Sequence.create({"name": "Putaway", "prefix": "PUT", "padding": 5}).id
+        picking_type_internal.write({
+                                        'sequence_id': sequence_putaway,
+                                        'sequence':13
+                                     })
+
+        location_path_vals = {
+            "name": "Putaway",
+            "route_id": route.id,
+            "sequence": 20,
+            "location_from_id": picking_type_internal.default_location_src_id.id,
```

Comment:
Not sure about this location, shouldn't it be goods-in's default_location_dest_id ?